tools/config.ts

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
please, can we use environment files for that to have the configuration for ci and local on one single place.

So we don't have to set paths with node environments:

`angular.json`
```json
"configurations": {
  "ci": {
    "fileReplacements": [
      {
        "replace": "tools/barista/src/environments/environment.ts",
        "with": "tools/barista/src/environments/environment.ci.ts"
      }
    ],
...
```

And on the ci you run it with `ng build barista-tools --configuration=ci`